public/Website/index.html

```diff
@@ -0,0 +1,29 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/bootstrap.css"
```

Comment:
import bootstrap from CDN. Don't download it

Reply:
https://getbootstrap.com/docs/5.0/getting-started/introduction/
for reference look into the quick start section